doc/configuration.rst

```diff
@@ -267,6 +267,19 @@ worker-wait-jitter
   Default: 5.0
 
 
+[worker]
```

Comment:
I can move the rest of the `worker-*` configs into this section in another PR if you'd like!